client/main.lua

```diff
@@ -246,8 +243,8 @@ function GiveKeys(id, plate)
 end
 
 function GetKeys()
-    QBCore.Functions.TriggerCallback('qb-vehiclekeys:server:GetVehicleKeys', function(keysList)
-        KeysList = keysList
+    lib.callback('qb-vehiclekeys:server:GetVehicleKeys', function(keysList)
```

Comment:
```suggestion
    lib.callback('qbx-vehiclekeys:server:GetVehicleKeys', function(keysList)
```
Since this is a callback it should be renamed to avoid confusion